File: alexandria/ui/src/io/intino/alexandria/ui/displays/Editor.java
package io.intino.alexandria.ui.displays;

import io.intino.alexandria.core.Box;
import io.intino.alexandria.ui.displays.notifiers.EditorNotifier;

public class Editor<DN extends EditorNotifier, B extends Box> extends Display<DN, B> {

	public Editor(B box) {
		super(box);
	}

}

File: alexandria/ui-elements/src/displays/Editor.js
import React from "react";
import AbstractEditor from "../../gen/displays/AbstractEditor";
import EditorNotifier from "../../gen/displays/notifiers/EditorNotifier";
import EditorRequester from "../../gen/displays/requesters/EditorRequester";

export default class Editor extends AbstractEditor {

	constructor(props) {
		super(props);
		this.notifier = new EditorNotifier(this);
		this.requester = new EditorRequester(this);
	};

	render() {
		return (
			<React.Fragment>
			</React.Fragment>
		);
	};


}